neuralmonkey/config/parsing.py

```diff
@@ -143,84 +142,49 @@ def parse_value(string):
 
     return string
 
+def parse_config(config_file, filename=""):
```

Comment:
To je složitější problém, který se mi nechtělo řešit, protože zasahuje minimálně do dalších tří submodulů. Nový `parse_config` skutečně parsuje konfig. `parse_file` dělá (i v původní verzi) něco jiného, než bylo v docstringu napsáno a aby se to spravilo, bylo by potřeba refaktorovat celý `neuralmonkey.config.*`.